dbt/adapters/contracts/relation.py

```diff
@@ -49,10 +49,12 @@ def __delitem__(self, key):
 
 class RelationConfig(Protocol):
     name: str
+    description: str
     database: str
     schema: str
     identifier: str
     compiled_code: Optional[str]
+    meta: Dict
```

Comment:
this could be `Dict[str, Any]`

If you wanted to add tags and resource_type here, I can close out https://github.com/dbt-labs/dbt-adapters/pull/120 in favor of this PR!

Reply:
Updated and added, closed #120